packages/website/sidebars.ts

```diff
@@ -124,23 +124,29 @@ const sidebars: SidebarsConfig = {
           "libraries/http/multipart",
           "libraries/http/encoding",
         ]),
-        createLibraryReferenceStructure("json-schema", "JSON Schema", false, []),
         createLibraryReferenceStructure("rest", "Rest", false, [
           "libraries/rest/cheat-sheet",
           "libraries/rest/resource-routing",
         ]),
         createLibraryReferenceStructure("openapi", "OpenAPI", false, []),
+        createLibraryReferenceStructure("versioning", "Versioning", false, [
+          "libraries/versioning/guide",
+        ]),
+        createLibraryReferenceStructure("xml", "Xml", false, ["libraries/xml/guide"]),
+      ],
+    },
+    {
+      type: "category",
+      label: "🖨️ Emitters",
+      items: [
+        createLibraryReferenceStructure("json-schema", "JSON Schema", false, []),
```

Comment:
Should we change to be in `emitters/` folder or keep all libraries in `libraries/`